index.html

```diff
@@ -234,7 +234,7 @@ <h1>Uber Navigation</h1>
                 </li>
               </ul>
             </div>
-            <button type="button" class="pro1">See Project</button>
+            <button type="button" class="pro1" id="3">See Project</button>
           </div>
           <div class="right-image">
             <img class="snap2" src="images3/desktop-project4.png" alt="desktop2">
```

Comment:
- [ ] Kindly as asked by the previous review you have to delete the whole `project` section from your HTML file, and create it by JS when the page loads, as required by the requirements `You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.`